docker/README.md

```diff
@@ -21,7 +21,7 @@ The scripts are:
 A base image for each OS is built using:
 
     cd docker/$os
-    docker build -t orc-$os .
+    docker build -t "orc-$os-jdk8" --build-arg jdk=8 . OR docker build -t "orc-$os-jdk11" --build-arg jdk=11 .
```

Comment:
Is this change easy to understand?

Reply:
Lets make it more explicit.
For jdk8:  docker build -t "orc-$os-jdk8" --build-arg jdk=8 .
For jdk11: docker build -t "orc-$os-jdk11" --build-arg jdk=11 .